neos/flow-development-collection
# FEATURE: Add kickstarter command to create basic XLIFF translation files

**What I did**

- Added a command for creating initial XLIFF translation files

**How I did it**

- Enhanced the Kickstarter 

**How to verify it**

Call `./flow kickstart:translation --package-key Neos.Demo --language-key de` and check the created folders and files at `Packages/Sites/Neos.Demo/Resources/Private/Translations`.

**Checklist**

 
Implements #880
